modules/clear_vacant_players.lua

```diff
@@ -142,36 +166,40 @@ function Public.set_offline_players_surface_removal(value)
 end
 
 --- Sets the active surface for this module, needs to be set else it will fail
----@param value string
+---@param value number|string Active surface index. Name of surface will also work.
 function Public.set_active_surface_index(value)
     this.settings.active_surface_index = value or nil
 end
 
+function Public.reset()
+    Public.clear_offline_players();
+end
+
 --- Clears the offline table
 function Public.clear_offline_players()
     this.offline_players = {}
 end
 
-local remove_offline_players = Public.remove_offline_players
+local function should_ignore_surface(surface)
```

Comment:
Is this supposed to be here? It looks like an empty function that is missing the ``end`` statement

Reply:
No, it's an artifact from rebasing/re-sorting.
It should be only in 8a8b52ab. :/

Fixed in 695a513e.